__tests__/app.test.js

```diff
@@ -64,6 +64,20 @@ describe('/api/categories', () => {
                 })
             })
         })
+        describe('GET /api/users', () => {
+            test('Happy path', () => {
+                return request(app).get('/api/users').expect(200).then(({body}) => {
+                    expect(body.length).not.toBe(0) 
```

Comment:
With regards testing the amount of user that will be returned from this endpoint, I think it would be more thorough to specifically test how many we expect to get back. If for some reason that a request to this endpoint only sent back half of the users, this test would still pass